src/routes/projects/[slug]/+page.server.ts

```diff
@@ -1,17 +1,19 @@
-import { contentWrapper } from "$lib/content/contentful";
 import { error } from "@sveltejs/kit";
 import type { Project } from "src/lib/utils/schema";
 import type { PageServerLoad } from "./$types";
 
 // Prerender all projects found by SvelteKit crawler (displayed in projects page)
 // But include server-side JS to load any projects not found (preview projects)
-// TODO: Actually hook into Contentful preview API by changing contentWrapper
 export const prerender = "auto";
 
-export const load: PageServerLoad = async ({ params }) => {
-  const projects: Project[] = await contentWrapper.get("project", {
-    "fields.slug": params.slug,
-  });
+export const load: PageServerLoad = async ({ params, locals }) => {
+  const projects: Project[] = await locals.contentWrapper.get(
+    "project",
+    {
+      "fields.slug": params.slug,
+    },
+    { preview: true }
```

Comment:
So is this the line that basically says that preview is supported on this route? And is that all the flag does?

Reply:
Yes, I see now it's a little unclear. I'll make a comment in the ContentWrapper to denote this. In short: "preview: true" just says, "use the preview Contentful API IF they supplied the secret query arg."

Reply:
Actually, one nit now that I looked at the relevant code: I would change this name to `allowPreview` to indicate that the presence of this flag being true doesn't mean we're in preview mode by default